common/logger.py

```diff
@@ -55,14 +55,16 @@ def warning(msg , parent = None, traceDepth = 0):
 def info(msg , parent = None, traceDepth = 0):
     if DEBUG:
         msg = '%s %s' %(_debugHeader(parent, traceDepth), msg)
-    print('%sINFO%s: %s' %(bcolors.OKGREEN, bcolors.ENDC, msg), file=sys.stdout)
+    print('%sINFO%s: %s' %(bcolors.OKGREEN, bcolors.ENDC, msg), file=sys.stderr)
     for line in tools.wrapLine(msg):
         syslog.syslog(syslog.LOG_INFO, 'INFO: ' + line)
 
 def debug(msg, parent = None, traceDepth = 0):
     if DEBUG:
```

Comment:
I would revert that if.

```
def debug()
    if not DEBUG:
        return
```

In that case you can save the indention of the following lines.
Clearer code IMHO.

Reply:
I think this would change the existing logging logic (different output) so I refrain from that.